### PR TITLE
Opal 684

### PR DESCRIPTION
**Pull request by JN-USAF**

### CODE REVIEW

File: weave/config.py [hunk 1]
```diff
@@ -98,8 +98,16 @@ def get_index_column_names():
 
 def get_file_system():
     """Get the filesystem to be used for storing baskets."""
+    # return s3fs.S3FileSystem(
```
Comment (JA-USAF) [inline weave/config.py:101]: Delete this if its not being used anymore

Comment (JN-USAF) [reply weave/config.py]: Thanks, it's removed

File: weave/config.py [hunk 2]
```diff
@@ -98,8 +98,14 @@ def get_index_column_names():
 
 def get_file_system():
     """Get the filesystem to be used for storing baskets."""
+    if "JUPYTERHUB_USER" in os.environ:
+        return s3fs.S3FileSystem(
+            client_kwargs={"endpoint_url": os.environ["S3_ENDPOINT"]}
+        )
     return s3fs.S3FileSystem(
```
Comment (JR-USAF) [inline weave/config.py:105]: Maybe use an environment variable "DISABLE_SSL"

if "DISABLE_SSL" in os.environ:

**Review by JR-USAF — CHANGES_REQUESTED** [review]

**Comment by JN-USAF** [comment]
These changes are required for running a local demo, which is a one time thing, so they should not be put into main.

Instead of merging into main, I'm going to rename this branch to demo-aug-23-2024.

**Pull request closed without merging**